Description: Fix spelling
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-01-20

--- python-falcon-0.3.0.orig/falcon/request.py
+++ python-falcon-0.3.0/falcon/request.py
@@ -144,7 +144,7 @@ class Request(object):
             where -1 is the last byte, -2 is the second-to-last byte,
             and so forth.
 
-            Only continous ranges are supported (e.g., "bytes=0-0,-1" would
+            Only continuous ranges are supported (e.g., "bytes=0-0,-1" would
             result in an HTTPBadRequest exception when the attribute is
             accessed.)
         if_match (str): Value of the If-Match header, or ``None`` if the
--- python-falcon-0.3.0.orig/falcon/response.py
+++ python-falcon-0.3.0/falcon/response.py
@@ -180,7 +180,7 @@ class Response(object):
             secure (bool): Direct the client to use only secure means to
                 contact the origin server whenever it sends back this cookie
                 (default: ``True``). Warning: You will also need to enforce
-                HTTPS for the cookies to be transfered securely.
+                HTTPS for the cookies to be transferred securely.
             http_only (bool): Direct the client to only transfer the cookie
                 with unscripted HTTP requests (default: ``True``). This is
                 intended to mitigate some forms of cross-site scripting.
