Description: Fix spelling
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-11

Index: python-falcon/falcon/request.py
===================================================================
--- python-falcon.orig/falcon/request.py
+++ python-falcon/falcon/request.py
@@ -189,7 +189,7 @@ class Request(object):
             where -1 is the last byte, -2 is the second-to-last byte,
             and so forth.
 
-            Only continous ranges are supported (e.g., "bytes=0-0,-1" would
+            Only continuous ranges are supported (e.g., "bytes=0-0,-1" would
             result in an HTTPBadRequest exception when the attribute is
             accessed.)
         range_unit (str): Unit of the range parsed from the value of the
